odiglet/odiglet.go

```diff
@@ -175,3 +179,27 @@ func runDeviceManager(clientset *kubernetes.Clientset, otelSdkLsf instrumentatio
 	manager.Run()
 	return nil
 }
+
+func OdigletInitPhase() {
+	if err := log.Init(); err != nil {
+		panic(err)
+	}
+	err := fs.CopyAgentsDirectoryToHost()
+	if err != nil {
+		log.Logger.Error(err, "Failed to copy agents directory to host")
+		os.Exit(-1)
+	}
+
+	nn, ok := os.LookupEnv(k8sconsts.NodeNameEnvVar)
+	if !ok {
+		log.Logger.Error(fmt.Errorf("env var %s is not set", k8sconsts.NodeNameEnvVar), "Failed to load env")
+		os.Exit(-1)
+	}
+
+	if err := k8snode.AddLabelToNode(nn, k8sconsts.OdigletInstalledLabel, k8sconsts.OdigletInstalledLabelValue); err != nil {
```

Comment:
I wonder if we should specify here if it's community or enterprise.
Since these 2 options will end up with different content in the node fs, and if a community pod get's scheduled on onprem node or vice-versa, it will crash the pod.